frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/CurrentPicker/periodPopoverText.unit.spec.js

```diff
@@ -0,0 +1,36 @@
+import { periodPopoverText } from "./periodPopoverText";
+
+describe("periodPopoverText", () => {
+  const now = new Date("2020-05-13T11:00:00.000Z");
+  Date.now = jest.fn().mockReturnValue(now);
```

Comment:
Please use `jest.setSystemTime`, like this https://github.com/metabase/metabase/blob/00096b412b67f9cef8fc6436a7ac242936aee262/enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationReviewIcon/ModerationReviewIcon.unit.spec.tsx#L15